assets/js/core.js

```diff
@@ -25,8 +25,7 @@ if(sessionStorage.userRoles && sessionStorage.userRoles.match(/Program Manager|S
   admin_tab_content += '<button class="overview-btns overview-btns-2nd-class" id="view-user" onclick="redirect(this.id); "><img src="/assets/images/edit-user.png" class="btn-icons"/><span>View user</span></button>';
 
   admin_tab_content += '<button class="overview-btns overview-btns-2nd-class" id="view-sys-settings" onclick="redirect(this.id); "><img src="/assets/images/sys-setting.png" class="btn-icons"/><span>System settings</span></button>';
-  
-  if(parseInt(sessionStorage.programID) == 1){
+  if(parseInt(sessionStorage.programID) == 1 || parseInt(sessionStorage.programID) == 14){
```

Comment:
This will also show Drug management, please refactor this to avoid showing that

Reply:
been resolved